apps/gitness/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@harnessio/canary": "workspace:*",
-    "@harnessio/code-service-client": "3.2.0",
+    "@harnessio/code-service-client": "3.2.1-beta.1",
```

Comment:
if the new code-service-client works correctly, let's remove the beta version, and publish 3.2.1 directly